### PR TITLE
Add example script for writing all Myo data to file.

### PR DESCRIPTION
**Pull request by jamieforth**

### CODE REVIEW

File: examples/myo-record.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import argparse
```
Comment (qtux) [inline examples/myo-record.py:8]: Could you please add a MIT license header with your credentials to the top of the file (see other files for an example)?

File: examples/myo-record.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+imu_header = ['timestamp', 'ori_w', 'ori_x', 'ori_y', 'ori_z', 'accel_1',
+              'accel_2', 'accel_3', 'gyro_1', 'gyro_2', 'gyro_3']
+
+
+class Myo(MyoRaw):
+
+    def __init__(self,  tty=None):
+        MyoRaw.__init__(self, tty)
+
```
Comment (qtux) [inline examples/myo-record.py]: The Myo class seems unnecessary to me. Is it here to give an example on how to implement a custom Myo class deriving from MyoRaw? I think it is better to remove the class and keep the example concise.

File: examples/myo-record.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                        help="Directory to write result files.")
+    args = parser.parse_args()
+
+    # Make output files.
+    now = datetime.fromtimestamp(time.time()).isoformat(timespec='seconds')
+    outdir = Path(args.outdir)
+    outdir.mkdir(parents=True, exist_ok=True)
+    emg_file = open(str(outdir / (now + '_emg.csv')), 'w', newline='')
+    imu_file = open(str(outdir / (now + '_imu.csv')), 'w', newline='')
+
```
Comment (qtux) [inline examples/myo-record.py]: Could you please use os.path.join to create the path, e.g. os.path.join(outdir, now + '_emg.csv')

File: examples/myo-record.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    emg_writer = csv.writer(emg_file, csv.unix_dialect,
+                            quoting=csv.QUOTE_MINIMAL)
+    emg_writer.writerow(emg_header)
+
+    imu_writer = csv.writer(imu_file, csv.unix_dialect,
+                            quoting=csv.QUOTE_MINIMAL)
+    imu_writer.writerow(imu_header)
+
+    m = Myo(args.tty)
```
Comment (qtux) [inline examples/myo-record.py]: As mentioned above: I prefer MyoRaw here instead of using the custom Myo class.

**Review by qtux — CHANGES_REQUESTED** [review]
Thank you for your nice example! I added some comments with minor issues. Could you please fix them?

File: examples/myo-record.py (hunk 1 above)
Comment (jamieforth) [reply examples/myo-record.py:8]: Added.

File: examples/myo-record.py (hunk 3 above)
Comment (jamieforth) [reply examples/myo-record.py]: As outdir was already a Path I used Path.joinpath, hope that's OK.

File: examples/myo-record.py (hunk 2 above)
Comment (jamieforth) [reply examples/myo-record.py]: Yes there's no need for the custom class here!

File: examples/myo-record.py (hunk 3 above)
Comment (qtux) [reply examples/myo-record.py]: Looks good to me.

**Review by qtux — APPROVED** [review]

**Comment by qtux** [comment]
Thank you very much for your nice example! It is working fine :-).

**Pull request merged**